static/app/views/issueList/actions/index.tsx

```diff
@@ -47,6 +48,13 @@ type IssueListActionsProps = {
   onActionTaken?: (itemIds: string[], data: IssueUpdateData) => void;
 };
 
+const animationProps = {
+  initial: {translateY: 10, opacity: 0},
+  animate: {translateY: 0, opacity: 1},
+  exit: {translateY: 10, opacity: 0},
```

Comment:
Let's try -10 on the exit. i think it will look cleaner

Reply:
Tried it out and I agree, does look a bit more natural!